src/modules/edc-demo/components/catalog-browser/catalog-browser.component.html

```diff
@@ -38,16 +38,41 @@
 
     <!-- Pagination -->
     <mat-paginator
+      *ngIf="contractOffersList.isReady"
       hidePageSize="true"
       pageSize="filteredContractOffers.length"
-      [length]="filteredContractOffers.length">
+      [length]="contractOffersList.data.filteredContractOffers.length">
     </mat-paginator>
   </div>
-
-  <edc-demo-contract-offer-list
-    *ngIf="filteredContractOffers.length"
-    [contractOffers]="filteredContractOffers"
-    (contractOfferClick)="
-      onContractOfferClick($event)
-    "></edc-demo-contract-offer-list>
+  <div class="flex flex-row justify-center uppercase">
```

Comment:
The uppercase class here is right now making empty messages uppercase.

It should be removed here and empty state / error state should specify such styling.